fhir-persistence/src/main/java/com/ibm/fhir/persistence/helper/FHIRTransactionHelper.java

```diff
@@ -88,4 +88,17 @@ public void rollback() {
             }
         }
     }
+
+    public void enroll() throws FHIRPersistenceException {
```

Comment:
javadoc?

Reply:
added description in the interface definition, didn't want to duplicate it..

Reply:
the issue is this doesn't implement the interface so it needs its own javadoc

Reply:
OK, let me add

Reply:
done